apps/voting/app/src/App.js

```diff
@@ -244,9 +245,16 @@ class App extends React.Component {
                   </Title>
                 }
                 endContent={
-                  <Button mode="strong" onClick={this.handleCreateVoteOpen}>
-                    New Vote
-                  </Button>
+                  <React.Fragment>
+                    <BreakPoint to="medium">
+                      <NewVoteButton onClick={this.handleCreateVoteOpen} />
```

Comment:
We should add a title:

```suggestion
                      <NewVoteButton
                        title="New Vote"
                        onClick={this.handleCreateVoteOpen}
                      />
```